libs/platform/src/lib/smart-filter-bar/components/smart-filter-bar-condition-field/smart-filter-bar-condition-field.component.html

```diff
@@ -0,0 +1,15 @@
+<ng-container [formGroup]="form">
+    <ng-container [formGroupName]="formGroupName">
+        <fdp-multi-input
+            [addOnButtonClickFn]="openConditionsDialog"
```

Comment:
when clicking on this, I get 
<img src="https://user-images.githubusercontent.com/33101123/148201102-419399c9-2c18-454e-91c8-dca6d880bdba.png" height="400px">

search doesn't really work
![image](https://user-images.githubusercontent.com/33101123/148201232-402dd52b-ed24-4623-8a72-dcb2426f886b.png)

Reply:
Should be fixed now. Please re-check.